doc/playground.lua

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env tarantool
+
+-- How to run:
```

Comment:
We need a 'How-to run samples' section in README.md

Reply:
And add `customers` space too, to make examples from README.md workable. And add README.md examples into `help()`. Okay.

Reply:
I have added the `customers` space, but don't add a section into readme. I'll do after PR #291.

Reply:
Added 'Quickstart' section into readme.